dali/base/dasds.cpp

```diff
@@ -1956,6 +1967,8 @@ class CExtCache
         if (mapIt != extTable.end())
         {
             auto listIter = mapIt->second;
+            Linked<CTransactionItem> &existingItem = *listIter;
+            cachedSz -= existingItem->ext.dataLength;
```

Comment:
@ghalliday - I spotted this in some late testing. Basically it would mean that although the cache was low, the cachedSz was left high, resulting in repeated spurious premature attempts to purge the ext. cache. It would mean that the external cache was effectively a 1-element cache after running for some time.